tests/python_package_test/test_dask.py

```diff
@@ -446,11 +446,29 @@ def test_classifier_pred_contrib(output, task, cluster):
                 assert len(np.unique(preds_with_contrib[:, base_value_col]) == 1)
 
 
-def test_find_random_open_port(cluster):
+def test_group_workers_by_host():
+    hosts = [f'0.0.0.{i}' for i in range(2)]
+    workers = [f'tcp://{host}:{p}' for p in range(2) for host in hosts]
+    expected = {
+        host: lgb.dask.HostWorkers(
```

Comment:
```suggestion
        host: lgb.dask._HostWorkers(
```

Sorry, CI is failing because I forgot this one.

Reply:
Haha yes, just saw that. Forgot about that test